File: Python/Easy/1.py
# Challenge: Calculate the Area of a Triangle
# Level: Beginner
# Description: This challenge involves creating a program that calculates the area of a triangle given its base and height.

# Instructions:
# 1. Ask the user to enter the base and height of the triangle.
# 2. Create a function that takes the base and height as arguments.
# 3. Inside the function, multiply the base by the height and divide the result by 2 to calculate the area of the triangle.
# 4. Return the area of the triangle.

# Example:
# Base: 5
# Height: 3
# Triangle Area: 7.5

# Function to calculate triangle area
def calculate_triangle_area(base, height):
    area = (base * height) / 2
    return area

# Asking user for input
base = float(input("Enter the base of the triangle: "))
height = float(input("Enter the height of the triangle: "))

# Calculating and displaying the area
triangle_area = calculate_triangle_area(base, height)
print(f"Triangle Area: {triangle_area}")

File: PHP/Easy/1.php
<?php
// Challenge: Check if a Word is a Palindrome
// Level: Beginner
// Description: This challenge involves creating a PHP function that checks if a word is a palindrome, meaning it reads the same forwards and backwards.

// Instructions:
// 1. Create a function called is_palindrome that takes a word as a parameter.
// 2. Inside the function, use the strrev() function to reverse the word.
// 3. Compare the original word with the reversed word using the strcasecmp() function to ignore case differences.
// 4. If the words are equal, return true, indicating that the word is a palindrome. Otherwise, return false.

// Example:
// Word: "reconocer"
// Is a palindrome: true

// Function to check if a word is a palindrome
function is_palindrome($word) {
    // Reverse the word using strrev
    $reversed_word = strrev($word);

    // Compare the original word with the reversed word, ignoring case differences
    return strcasecmp($word, $reversed_word) === 0;
}

// Example usage:
$userWord = readline("Enter a word: ");
$isPalindrome = is_palindrome($userWord);

if ($isPalindrome) {
    echo "Is a palindrome: true\n";
} else {
    echo "Is a palindrome: false\n";
}
?>
